config.c

```diff
@@ -0,0 +1,294 @@
+/**
+ * Config Reader and Parser
+ * This program will read the config for lightwm in the appdata directory
+ * 
+ * Demetry Romanowski
+ * demetryromanowski@gmail.com
+ **/ 
+#include "config.h" 
+
+#include <Windows.h> 
+#include <Shlobj.h> 
+#include <wchar.h> 
+#include <stdio.h> 
+#include <strsafe.h> 
+#include <shlwapi.h> 
+#include <assert.h> 
+
+#include "error.h" 
+#include "resource.h"
+
+#include "debug.h"
+
+#define BUFF_SIZE 65536
+
+/**
+ * Config reader global vars
```

Comment:
(Regarding this comment and all the rest) By convention I prefer to minimize comments in the code, especially obvious ones. Only when the comment has real added value, I like adding them - but that is rare.